docs: Building Simulation Methodology (#27)
PassiveLogic/differentiable-swift-examples: Benchmarks/BuildingSimulation/README.md

    <td>2278656</td>
    <td>1.1</td>
    <td>133253440</td>
    <td>62</td>
    <td>236719872</td>
    <td>109</td>
  </tr>
  <tr>
    <td>100000</td>
    <td>2098368</td>
    <td>2409728</td>
    <td>1.1</td>
    <td>134318336</td>
    <td>64</td>
    <td>240111552</td>
    <td>114</td>
  </tr>
  <tr>
    <td>N timesteps<br>trials=1<br>warmup=3</td>
    <td></td>
    <td></td>
    <td></td>
    <td></td>
    <td></td>
    <td></td>
    <td></td>
  </tr>
  <tr>
    <td>100</td>
    <td>1934528</td>
    <td>2393280</td>
    <td>1.2</td>
    <td>143346176</td>
    <td>74</td>
    <td>238309248</td>
    <td>123</td>
  </tr>
  <tr>
    <td>1000</td>
    <td>2393280</td>
    <td>4064448</td>
    <td>1.7</td>
    <td>292129920</td>
    <td>122</td>
    <td>239030080</td>
    <td>99</td>
  </tr>
  <tr>
    <td>10000</td>
    <td>6964480</td>
    <td>23430464</td>
    <td>3.4</td>
    <td>1819930944</td>
    <td>261</td>
    <td>293621952</td>
    <td>42</td>
  </tr>
  <tr>
    <td>100000</td>
    <td>48531072</td>
    <td>214257152</td>
    <td>4.4</td>
    <td>17550355200</td>
    <td>361</td>
    <td>806886848</td>
    <td>16</td>
  </tr>
</tbody></table>

---
#### Power Consumption

> In order to measure the energy consumed during program execution we employed the use of a current shunt, a differential probe, and an oscilloscope.

Reviewer: ```suggestion
We measured energy consumption during program execution using a current shunt, a differential probe, and an oscilloscope.
```
